### DISCUSSION TITLE
How to filter search results?

### DISCUSSION THREAD

**Discussion by ghost** [TeamNewPipe/NewPipe#11437] [Questions] [answered]
### Checklist

- [X] I made sure that there are *no existing issues or discussions* - [open](https://github.com/TeamNewPipe/NewPipe/issues) or [closed](https://github.com/TeamNewPipe/NewPipe/issues?q=is%3Aissue+is%3Aclosed) - which I could contribute my information to.
- [X] I have read the [FAQ](https://newpipe.net/FAQ/) and my question isn't listed.
- [X] I have taken the time to fill in all the required details. I understand that the question will be dismissed otherwise.
- [X] I have read and understood the [contribution guidelines](https://github.com/TeamNewPipe/NewPipe/blob/dev/.github/CONTRIBUTING.md).

### What is/are your question(s)?

I think enabling the "show age restricted content" option is too restrictive, so I would like to manually modify the code to either:

1. Filter the search query
2. Filter the resulting videos by searching titles and/or description strings.

Where is the most suitable place to add such modifications please?

Thanks.

**Comment by asandikci** [comment, marked as answer]
May be related to: https://github.com/TeamNewPipe/NewPipe/pull/8837
